common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -606,6 +606,12 @@
     ],
     "sqlState" : "42711"
   },
+  "COLUMN_ARRAY_ELEMENT_TYPE_MISMATCH" : {
+    "message" : [
+      "Some values in field <pos> are incompatible with the column array type. Expected type <type>."
+    ],
+    "sqlState" : "0A000"
```

Comment:
Why did you select this error class:
```
    "0A": "feature not supported",
```
Please, consider another one, for instance some sqlState with the error class `42`

Reply:
Please, assign a `sqlState` of the error class `42`.